PowerShellTools.HostService/ServiceCommon.cs

```diff
@@ -16,6 +16,7 @@ namespace PowerShellTools.HostService
 {
     public class ServiceCommon
     {
+        public static object RunspaceLock = new object();

```

Comment:
Can this be internal?

[Also, a TODO to refactor this to be wholly contained within the HostService itself, so if host is ever decoupled from VS, we won't have to worry as much about it?]